import-service/package.json

```diff
@@ -17,6 +16,7 @@
     "@types/jest": "^26.0.15",
     "@types/node": "^10.12.18",
     "@types/serverless": "^1.72.5",
+    "aws-sdk": "^2.794.0",
```

Comment:
Do we need this dependency in lambda's package.json? I think AWS has these features available without downloading the sdk.